build/build-image/cross/Dockerfile

```diff
@@ -62,14 +62,13 @@ RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/
 
 # work around 64MB tmpfs size in Docker 1.6
 ENV TMPDIR /tmp.k8s
-
-# Get the code coverage tool, goimports, and godep
 RUN mkdir $TMPDIR \
   && chmod a+rwx $TMPDIR \
-  && chmod o+t $TMPDIR \
-  && go get golang.org/x/tools/cmd/cover \
-            golang.org/x/tools/cmd/goimports \
-            github.com/tools/godep
+  && chmod o+t $TMPDIR
+
+# Get the code coverage tool and goimports
```

Comment:
it looks like a few generated code tools use `goimports` still.

`hack/make-rules/test.sh` has a coverage mode. I'm not sure how many people still use it. I really don't know how many people use it in combination with the build image.